core/server/api/v2/utils/serializers/output/utils/extra-attrs.js

```diff
@@ -0,0 +1,12 @@
+module.exports.forPost = (frame, model, attrs) => {
+    const _ = require('lodash');
+
+    if (!frame.options.hasOwnProperty('columns')) {
+        if (_.isEmpty(attrs.custom_excerpt)) {
+            const plaintext = model.get('plaintext');
+            attrs.excerpt = plaintext.substring(0, 500);
+        } else {
+            attrs.excerpt = attrs.custom_excerpt;
+        }
+    }
```

Comment:
@ErisDS I've simplified the logic. We can access the plaintext via the model. Forgot about that 🤪I thought we don't even fetch the plaintext field if we only return html by default.